### PR TITLE
ADEN-13047 - webads/relevantdigital bidder integration

### PR DESCRIPTION
**Pull request by adi-wesolowski-fandom**
jira: https://fandom.atlassian.net/browse/ADEN-12216

### CODE REVIEW

File: src/ad-tracking/compilers/slot-bidders-tracking-compiler.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ async function getBiddersPrices(
 		bidder_33: transformBidderPrice('criteo'),
 		bidder_34: transformBidderPrice('yahoossp'),
 		bidder_35: transformBidderPrice('freewheel'),
+		bidder_36: transformBidderPrice('webAds'),
```
Comment (Jorewin) [inline src/ad-tracking/compilers/slot-bidders-tracking-compiler.ts:45]: We got ourselves a new bidder recently and it's configured in [crossbow](https://github.com/Wikia/crossbow/commit/d86d5875dd6895353181be8a4b6674194912d266). It is set there as `bidder_36` so the new one should be probably `bidder_37`. Do you know why the new one one in't here?

File: src/platforms/shared/bidders/bidders-state.setup.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export class BiddersStateSetup implements DiProcess {
 		triplelift: 'icPrebidTriplelift',
 		verizon: 'icPrebidVerizon',
 		yahoossp: 'icPrebidYahooSsp',
+		relevantdigital: 'icWebAds',
```
Comment (Jorewin) [inline src/platforms/shared/bidders/bidders-state.setup.ts:29]: Shouldn't this be like that?
```suggestion
		relevantdigital: 'icPrebidWebAds',
```
Also, why `relevantdigital`?

File: src/platforms/ucp-mobile/bidders/prebid/webads.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	return {
+		enabled: false,
+		slots: {
+			top_leaderboard: {
+				sizes: [
+					[320, 50],
+					[320, 100],
+				],
+				placementId: '648079843898112402b6dfdc_647766970ee778cdfd3e1d66',
+				accountId: '647765b7705d4fca3b3e1d58',
```
Comment (Jorewin) [inline src/platforms/ucp-mobile/bidders/prebid/webads.ts:11]: Will the `accountId` ever change/be different? Maybe we can hard-code it in the adapter?

File: src/platforms/shared/bidders/bidders-state.setup.ts (hunk 2 above)
Comment (lnawrotek) [reply src/platforms/shared/bidders/bidders-state.setup.ts:29]: It's the name in Prebid library, we use the name WebAds for our purpose.

File: src/ad-tracking/compilers/slot-bidders-tracking-compiler.ts (hunk 1 above)
Comment (lnawrotek) [reply src/ad-tracking/compilers/slot-bidders-tracking-compiler.ts:45]: I've changed. I don't know why 36 is missing

File: src/platforms/shared/bidders/bidders-state.setup.ts (hunk 2 above)
Comment (Jorewin) [reply src/platforms/shared/bidders/bidders-state.setup.ts:29]: What about the suggested change? Just want to make sure you didn't miss it.

**Review by Jorewin — APPROVED** [review]

**Review by preludek — APPROVED** [review]

**Pull request merged**